src/libraries/navdrawer/navdrawer.js

```diff
@@ -2,6 +2,7 @@
  * and will be replaced soon by a Vue component.
  */
 
+/* eslint-disable no-var */
```

Comment:
Why disable the rule for this file?

Reply:
see the lines directly above, we should remove this file eventually